src/frontend/admin/tsconfig.json

```diff
@@ -1,7 +1,9 @@
 {
   "compilerOptions": {
-    "target": "es5",
-    "lib": ["dom", "dom.iterable", "esnext"],
+    "target": "ESNext",
```

Comment:
It seems that `ESNext` will be using `es2022` and I'm not sure every browser really support all the features of `es2022`  yet. To discuss in daily?